### ISSUE TITLE
Incorrect alert when draft product is modified

### ISSUE THREAD

**Issue by WPWhiteSecurity** [WPWhiteSecurity/WP-Security-Audit-Log#355]
Any change that is applied to a product that is saved as draft is reported as alert 9000 (created product). Changes for draft product should be the same as for published product, such as changed shorted description (9004), changed the text (9005) etc.

**Comment by WPWhiteSecurity** [comment]
This issue persists. Here are all the test cases:

modifying short description of draft shows alert 9000. It should show alert 9004
modifying text of draft shows alert 9000. It should show alert 9005
modifying URL of draft shows alert 9000. It should show alert 9006
modfying product data of draft shows alert 9000. It should show alert 9007
changing the date of a draft shows alert 9000. It should show alert 9008
changing the price of a draft shows alert 9000. It should show alert 9016
changing SKU of a draft shows  alert 9000. It should show alert 9017
changing the stock of a draft shows alert 9000. It should show alert 9018
changing the stock quantity of a draft shows alert 9000. It should show alert 9019
changing the product type of a draft shows alert 9000. It should show alert 9020
changing the weight of a draft shows alert 9000. It should show alert 9021
changing the dimensions of a draft shows alert 9000. It should show alert 9022
adding downloadable file to draft shows alert 9000. It should show alert 9023
removing downloadable file in draft shows alert 9000. It should show alert 9024
renaming downloadable file in draft shows alert 9000. It should show alert 9025
changing URL of downloadable file in draft shows alert 9000. It should show alert 9026

So basically the modification of data on a draft post should be the same as that of a published post.
